### PR TITLE
If in multihop, then only show city names in the CI page

### PR DESCRIPTION
**Pull request by Gela**
## Description

If in multihop, then only show city names in the CI page. This change does not affect the UI in single hop mode.

## Reference

    https://mozilla-hub.atlassian.net/browse/VPN-3639

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [x] I have added thorough tests where needed

### CODE REVIEW

File: nebula/ui/components/VPNConnectionInfoContent.qml
Comment (MattLichtenstein) [inline nebula/ui/components/VPNConnectionInfoContent.qml:123]: using anchors inside a component that is in a layout will usually cause issues and unintended affects (and of course a qml warning 😂)

Screen Shot 2023-01-11 at 4 28 19 PM

as far as how to resolve this... it sort of depends on what the intended design is. 

- Do we want the arrow to always be dead center, splitting the two labels 50/50?
- Do we want the arrow to move in case we have a longer city on one side than the other?
- If we truncate one city, do we always want to truncate the other (for consistency) or do we want to try to always have at least one of the cities not truncated

@peterbenvenuto any thoughts?

Comment (Gela) [reply nebula/ui/components/VPNConnectionInfoContent.qml:123]: Good point, I played around with some of the longest names I could find (NYC->NYC) and not seeing the same issue you mentioned. I wonder why in my case it adds "..." and in your it does not? 🤔 

I suppose I'm leaning towards not centering the arrow but let's see what @peterbenvenuto thinks.

Screenshot 2023-01-11 at 1 42 10 PM

File: nebula/ui/components/VPNConnectionInfoContent.qml
Comment (bakulf) [inline nebula/ui/components/VPNConnectionInfoContent.qml:138]: This is the only place where we use: `VPNServerCountryModel.getLocalizedCountryName`.
Can we get rid of it? I would like to see: `VPNCurrentServer.localizedCountryName` instead.
Do you mind to do this? As part of this PR or a separate one.

**Review by bakulf — APPROVED** [review]
I reviewed only the use of the model. Please, wait for a review from front-end experts before merging.

File: nebula/ui/components/VPNConnectionInfoContent.qml
Comment (Gela) [reply nebula/ui/components/VPNConnectionInfoContent.qml:138]: Jira issue: https://mozilla-hub.atlassian.net/browse/VPN-3884

**Pull request merged**
